gst/gst_element.go

```diff
@@ -175,6 +175,12 @@ func (e *Element) CallAsync(f func()) {
 	)
 }
 
+// PostMessage posts a message on the element's bus
+// Takes ownership of the message so need to ref it before feeding it forward
+func (e *Element) PostMessage(message *Message) bool {
+	return gobool(C.gst_element_post_message(e.Instance(), message.Ref().Instance()))
```

Comment:
I'd like to avoid any mentioning of ref and unrefs in the public function documentation, since that was the biggest source of issues before, because users kept calling `Ref` and `Unref` which caused memory leaks and segfauls.

```suggestion
// PostMessage posts a message on the element's bus
func (e *Element) PostMessage(message *Message) bool {
	// gst_element_post_message takes ownership of the message so need to ref it before feeding it forward
	return gobool(C.gst_element_post_message(e.Instance(), message.Ref().Instance()))
```